### ISSUE TITLE
Reevaluate the workflows regarding Default Token configuration to have a more clear design about when these settings are applied to actor creation workflows to better differentiate user intent.

### ISSUE THREAD

**Issue by mrkwnzl** [foundryvtt/foundryvtt#9818]
### What happened?

When I changed the Default Token configuration in the core settings, those settings overwrite the prototype token settings of actors that are being imported from compendiums. I expect that importing actors from compendiums respects the prototype token settings.

### What ways of accessing Foundry can you encounter this issue in?

- [X] Native App (Electron)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] #11950

### Reproduction Steps

Take the dnd5e system, change the default token configuration in the core settings. Import a monster from the monster compendium. Compare the prototype token settings with the imported actor with the ones from the compendium. They differ, although they shouldn’t.

### What core version are you reporting this for?

11.306

### Bug Checklist

- [X] The issue occurs while all Modules are disabled

**Comment by unsoluble** [comment]
This is expected behaviour, per #8660. You can override the defaults by holding Alt on an import drag.

**Comment by mrkwnzl** [comment]
@unsoluble Thanks!

Then this becomes a feature request to reverse that behavior. I think it would be more intuitive that the specific token settings overwrite the general settings for new tokens, while holding alt while importing reverts them to the default. Usually, there's a reason why the specific tokens get their settings, and reverting to the defaults defeats that purpose.

**Comment by mrkwnzl** [comment]
I played around with this and it’s actually quite confusing as it is now. If certain Default Token Configuration settings are in their default state, that default state is *not* applied when importing actors from a compendium. For example, if I set the Display Name to `Never Displayed` but have a prototype token set to `Hover for Owner`, when I import that actor from a compendium, that token will have the Display Name set to `Hover for Owner`, regardless of whether I hold alt or not. That means if I really want to default to `Never Displayed`, I can’t do that with imported actors right now.

Even if you decide against switching the alt function, the default settings should be applied to imported actors across the board. Either apply *all* the settings I set in the Default Token Configuration settings (and that includes the default values) or also import all prototype settings from the actors that are being imported.

Right now, this is a combination of applying only settings which are not the default settings from the Default Token Configuration and the ones already set on the actors prototype settings, so now there are cases where I neither get all my settings set in the Default Token Configuration nor all the settings from the actors. I don’t think that’s a great user experience.

Thanks for considering this!
